plugins/modules/notification/slack.py

```diff
@@ -309,13 +332,31 @@ def build_payload_for_slack(module, text, channel, thread_id, username, icon_url
     return payload
 
 
+def get_slack_message(module, domain, token, channel, ts):
+    headers = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
+        'Authorization': 'Bearer ' + token
+    }
+    url = SLACK_CONVERSATIONS_HISTORY_WEBAPI + '?channel=%s&ts=%s&limit=1&inclusive=true' % (channel, ts)
```

Comment:
is it worth importing `urllib.parse.urlencode` for this? I was surprised that `fetch_url` doesn't pass `params` to do it for us.

Reply:
I'd personally would prefer `urllib.parse.urlencode` (resp. the corresponding six import, I guess you can find it by grepping for `urlencode` in this repo) since it will make sure there are no problems if someone provides `channel` or `ts` with unexpected content (like `&` :) ).

Reply:
Just grepped for something related, here's the import you need: `from ansible.module_utils.six.moves.urllib.parse import urlencode`